Add a warning about the API (#1067)
timescale/docs: timescaledb/tutorials/analyze-nft-data/nft-schema-ingestion.md

key, see the [OpenSea API documentation](https://docs.opensea.io/reference/request-an-api-key).
</highlight>

> <highlight type="warning">
> The outlined process below only works if the OpenSea API works. This is
> outside of our control. If you want to make sure you can complete this
> tutorial smoothly we recommend [downloading the sample data](https://assets.timescale.com/docs/downloads/nft_sample.zip)
> and importing it.
> </highlight>

Reviewer: ```suggestion
<highlight type="warning">
The following procedure relies on the OpenSea API provided and maintained by OpenSea. Unfortunately,
there have recently been extended periods of time when the API is not functioning, something that is
outside of our control. 

If the API has changed or is not accessible when you attempt to run the `opensea_ingest.py`, we suggest following the second procedure to download a historical data file and importing it. You will be able to successfully 
complete the tutorial with that data alone.
</highlight>
```

key, see the [OpenSea API documentation](https://docs.opensea.io/reference/request-an-api-key).
</highlight>

<highlight type="warning">

> The following procedure relies on the OpenSea API provided and maintained by OpenSea. Unfortunately,
> there have recently been extended periods of time when the API is not functioning, something that is
> outside of our control. 
> 
> If the API has changed or is not accessible when you attempt to run the `opensea_ingest.py` script, we suggest following the second procedure to download a historical data file and importing it. You will be able to successfully 
> complete the tutorial with that data alone.

Reviewer 2: ```suggestion
This procedure relies on the OpenSea API. The OpenSea API is provided and 
maintained by OpenSea. Recently, the API is stopped functioning for extended 
periods of time. If the API has changed or is not accessible when you attempt 
to run the `opensea_ingest.py` script, try following the procedure to download 
an historical data file and import it. You can use this data file to complete the 
tutorial.
```
